cutil/initapi/validate.go

```diff
@@ -14,4 +14,22 @@
 
 package initapi
 
-// keep
+import (
+	"fmt"
+	"github.com/kris-nova/kubicorn/apis/cluster"
+)
+
+func serverPoolCounts(initCluster *cluster.Cluster) error {
```

Comment:
Certainly! How exactly would we use reflection to accomplish that? It all seems like dark magic to me, so a real-world example would be pretty helpful.